ibm/mas_devops/roles/uds_install/defaults/main.yml

```diff
@@ -1,4 +1,5 @@
 ---
+cluster_name: "{{ lookup('env', 'CLUSTER_NAME')}}"
```

Comment:
Please add to the role `README.md` info about this, e.g. that it's required on ROSA, but not used anywhere else

Reply:
@durera Made changes to the README.md as requested